src/containers/home/banner.tsx

```diff
@@ -28,33 +61,84 @@ const Banner = () => {
             <Row gutter={[16, { xs: 16, sm: 20, md: 27, lg: 27 }]}>
               <Col span={24}>
                 <Typography.Title level={1} className="title">
-                  An Open Liquidity Protocol on{' '}
+                  An Open <br /> Liquidity Protocol <br /> on{' '}
                   <strong className="gradient-text">Solana</strong>
                 </Typography.Title>
               </Col>
               <Col span={24} style={{ position: 'relative' }}>
                 <Typography.Text type="secondary" className="description">
-                  All-in-One Solana Open Platform with DApps Store and Universal
-                  Protocol for Liquidity.
+                  The #1 AMM built on Solana to enrich the proficiency of DeFi
+                  ecosystem.
                 </Typography.Text>
               </Col>
             </Row>
           </Col>
           <Col span={24}>
             <Row gutter={[16, 16]}>
-              <Col xs={24} md={16} lg={10}>
+              <Col xs={24} md={9} lg={8} xl={6}>
                 <Button
                   type="primary"
-                  size="middle"
+                  size="large"
                   block
                   onClick={() => window.open(SENTRE_URL, '_blank')}
+                  disabled={isDisable}
                 >
-                  Launch Sentre
+                  <span style={{ fontSize: '14px' }}>Launch Sentre</span>
                 </Button>
               </Col>
             </Row>
           </Col>
+          <Col span={24}>
+            <Typography.Paragraph className="btn-launch">
+              Official Sentre launch in
+            </Typography.Paragraph>
+          </Col>
+          <Col span={24}>
+            <Row gutter={{ xs: 44, sm: 28, md: 28, lg: 28, xl: 54 }}>
+              <Col xs={8} sm={8} md={8} lg={6}>
+                <div className="count-down">
+                  <p>{remainingTime.days}</p>
+                  <Typography.Text
+                    style={{ textTransform: 'uppercase', fontSize: '14px' }}
```

Comment:
Typography.Text font size default is 14px

Reply:
em bỏ ra nhưng nó k nhận fontsize 14px anh ơi